internal/frontend/default-frontend.go

```diff
@@ -541,6 +523,26 @@ func (self *defaultFrontend) getCmdEnvContext(envVars []string, consents []strin
 		}
 	}
 
+	/* append environment variables that do not requires consent */
```

Comment:
```suggestion
	/* append environment variables that do not require consent */
```
OPT

Reply:
Resolved in commit [c0eed5b](https://github.com/criteo/command-launcher/pull/149/commits/c0eed5b237e1256211c066fda3ce74fc7853f76c)